addins/Cake.Issues.Testing.yml

```diff
@@ -2,7 +2,7 @@ Name: Cake.Issues.Testing
 NuGet: Cake.Issues.Testing
 Assemblies:
 - "/**/Cake.Issues.Testing.dll"
-Repository: https://github.com/cake-contrib/Cake.Issues.Testing
+Repository: http://cake-contrib.github.io/Cake.Issues.Website/
```

Comment:
@Jericho this seems wrong to me?

Reply:
It was taken straight from the nuspec. See the "Project Site" link in [this page](https://www.nuget.org/packages/Cake.Issues.Testing/). You should raise an issue in the addin repo if you feel this link is wrong.

I happen to think this is actually great. It points to the addin's documentation.

Reply:
This is the correct project URL, but not the repository. So the question is what do we expect here, as these two things are different fields in the nuspec metadata

Reply:
the `Repository` property in our yaml expects the project's URL, not the git repo URL. I which the property was called something different, like ProjectUrl for example (which would match the name of the property in nuspec).

Reply:
Cake.Issues.Testing is not actually an addin, but infrastructure to help write tests for Cake.Issues.* addins. So it should probably be removed from here.

@Jericho Why is the URL only changed in this case and not for the other Cake.Issues addins, where the same site is set for the project site link?

Reply:
@Jericho If you check " Source repository " on
https://www.nuget.org/packages/Cake.Issues.Testing/
it points to https://github.com/cake-contrib/Cake.Issues

which feels correct to me.

Reply:
@devlead you are looking at the wrong nuspec value. The source repository is URL to the actual git repo. Our YAML expects URL to the "project" which is in the "Project Site" nuspec value.

Reply:
@pascalberger @devlead in fact, the "Source Repository" URL is not quite correct: it should include the ".git" extension.

Reply:
@pascalberger the audit tool simple gets this value from the nuspec. In this case, the value that was previously in the YAML file does not match the nuspec that's why the audit tool is proposing a change. I haven't reviewed your other addins so I don't know why similar modification was not proposed.

Reply:
@Jericho Should be the same for the other Cake.Issues addins. E.g. [Cake.Issues](https://www.nuget.org/packages/Cake.Issues/) has `https://cake-contrib.github.io/Cake.Issues.Website/` set as project site but [https://github.com/cake-contrib/Cake.Issues](https://github.com/cake-build/website/blob/master/addins/Cake.Issues.yml#L5) in the yaml.

Reply:
@pascalberger I just noticed something: the name of the addin is `Cake.Issues.Testing` but there is no repo with matching name under the cake-contrib. If I understand correctly, the name of the repo is `Cake.Issues`. The naming discrepency is probably causing the audit tool to be unable to figure out that this addin has been moved under the cake-contrib organization.

Reply:
Your other addins seem to follow the naming convention (repo name == addin name) and therefore the audit tool can figure out they have been move to cake-contrib.

Reply:
@pascalberger hummmm. The more I look at it, the more complicated the situation is getting. If you look at the [Excel spreadsheet generated by the audit tool](https://github.com/cake-contrib/Home/blob/master/Audit.md#additional-audit-results), you'll notice that the addin is correctly indicated as having been moved to cake-contrib. So there is something else going on with this addin. I'm continuing my investigation...

Reply:
ok I figured it out: the audit tool is able to figure out the project has been transferred to cake-contrib because of the presence of the RepositoryUrl in nuspec BUT the project URL is not set to 'https://github.com/cake-contrib/cake.issues.testing' because there is no such named github repo. That is the difference with your other Cake.Issues.* addins.